flipkart_scraper.py

```diff
@@ -0,0 +1,133 @@
+import requests  
+from urllib.request import urlopen as ureq
+from bs4 import BeautifulSoup as soup 
+import smtplib 
+import io  
+import csv 
+
+# The front end can make use of the passing parameters
+# In the dropdown list, the developer can give the Words like Laptops, Mobiles, Cameras, Headphones, Air conditioners etc
+# Also the user can define the limit of number of pages to scrap [It is used in both functions]
+
+def get_urls(DropDownSelection,limit):
+  '''The First function executed when scrape_page(get_urls(dds,limit),limit) is called
+  is the get_url, it makes a list of links according to the limit and selection
+  in order to forward to the scraper part'''
```

Comment:
The best way to use doctrings is the following format:

```python
def my_function(arg1): 
    """ 
    Summary line. 
  
    Extended description of function. 
  
    Parameters: 
    arg1 (int): Description of arg1 
  
    Returns: 
    int: Description of return value 
  
    """
  
    return arg1 
```

Also, "unlike conventional source code comments, the docstring should describe what the function does, not how". So here, it would be better if you describe in simple terms what `get_urls` takes as input and what it returns. Rest of the "good to know" things can be put in a comment somewhere else.